### DISCUSSION TITLE
Feature Request - Auto Scroll remote desktop

### DISCUSSION THREAD

**Discussion by BDaddyG** [rustdesk/rustdesk#1191] [Ideas]
Hello

I'd like to suggest the option to have the remote desktop automatically scroll as the mouse moves to the edges of the viewer window.

Situation:  Remote screen size is larger than the local screen (accessing desktop PC with large screen from a laptop with a small screen).  Have to either use "shrink" or "original".  

When using "shrink" everything is too small for me to see.  When using "original", I have to manually use the scrolls to get to different parts of the remote desktop.

At work I use Anydesk, migrating away from "Remote Utilities", both of which have this feature. I've grown to appreciate this and miss it when using RustDesk.

Thanks!

**Comment by rustdesk** [comment]
but I like scrollbar more. i do not like anydesk mode, the view always moves along with the mouse, it is annoying.

**Reply by BDaddyG** [reply]
Could it be an option in the preferences to enable or disable?

**Reply by rustdesk** [reply]
yes, what name should it be?

**Reply by BDaddyG** [reply]
Looks like it's called "edge scrolling in original view mode" in Anydesk, so "Edge Scrolling" works for me.

**Comment by dsm150** [comment]
it would be cool to provide a choice in the settings ...

**Reply by dsm150** [reply]
I think the colleague meant something similar.  If it were possible to implement such a scrolling option - it would be great!

Sample as it works:

https://youtube.com/shorts/tP9dnZZsXP0?feature=share

and, it's a good idea to be able to see both monitors at once, like in the example above
